simplex/test_util.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package simplex
+
+import (
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/snow/validators"
+	"github.com/ava-labs/avalanchego/utils/crypto/bls/signer/localsigner"
+)
+
+func newTestValidatorInfo(allVds []validators.GetValidatorOutput) map[ids.NodeID]*validators.GetValidatorOutput {
+	vds := make(map[ids.NodeID]*validators.GetValidatorOutput, len(allVds))
+	for _, vd := range allVds {
+		vds[vd.NodeID] = &vd
+	}
+
+	return vds
+}
+
+func newEngineConfig() (*Config, error) {
+	ls, err := localsigner.New()
+	if err != nil {
+		return nil, err
+	}
+
+	nodeID := ids.GenerateTestNodeID()
+
+	simplexChainContext := SimplexChainContext{
+		NodeID:   nodeID,
+		ChainID:  ids.GenerateTestID(),
+		SubnetID: ids.GenerateTestID(),
+	}
+
+	nodeInfo := validators.GetValidatorOutput{
+		NodeID:    nodeID,
+		PublicKey: ls.PublicKey(),
+	}
+
+	return &Config{
+		Ctx:        simplexChainContext,
+		Validators: newTestValidatorInfo([]validators.GetValidatorOutput{nodeInfo}),
+		SignBLS:    ls.Sign,
+	}, nil
+}
```

Comment:
If the helper function is used later, feel free to keep it... But it looks unnecessary in this PR.
```suggestion
func newEngineConfig() (*Config, error) {
	ls, err := localsigner.New()
	if err != nil {
		return nil, err
	}

	nodeID := ids.GenerateTestNodeID()
	return &Config{
		Ctx: SimplexChainContext{
			NodeID:   nodeID,
			ChainID:  ids.GenerateTestID(),
			SubnetID: ids.GenerateTestID(),
		},
		Validators: map[ids.NodeID]*validators.GetValidatorOutput{
			nodeID: {
				NodeID:    nodeID,
				PublicKey: ls.PublicKey(),
			},
		},
		SignBLS: ls.Sign,
	}, nil
}
```